src/Form/FormMapper.php

```diff
@@ -136,9 +136,19 @@ public function add($name, $type = null, array $options = [], array $fieldDescri
                 $options['label'] = $this->admin->getLabelTranslatorStrategy()->getLabel($name, 'form', 'label');
             }
 
+            // NEXT_MAJOR: Remove this block.
             if (isset($options['help'])) {
-                $fieldDescription->setHelp($options['help']);
-                unset($options['help']);
```

Comment:
Symfony `help` option didn't work because the option was removed.

Reply:
**description** option

Reply:
What do you mean?

Reply:
Truly says was:
> Symfony help option didn't work because the **description option** was removed.

Reply:
hmm not exactly true, here at `FormMapper` it checked if the `help` form option was set, given:

```php
->add('field', null, ['help' => 'message'])
```
the `$options` variable contains among other things `['help' => 'message']`, then it executed:

`$fieldDescription->setHelp($options['help']);` so this help message is set as field description option and then:

`unset($options['help']);` which removes the **form option**, so it's never in the `$options` array that is passed to the form builder.

Reply:
After you fix `help` dosnt unset from `options`

Reply:
Dont miss where options and field description options

Reply:
You says:
> **Symfony help** option didn't work because the option was removed.
And
>  Use Symfony Form "help" option instead.

This is mismatch statements

Reply:
Ok... I added that comment at the time I made the PR to explain the reason behind that change, so what I meant is... **before** making this change, Symfony Form "help" option didn't work as you expected (using `->add('field', null, ['help' => 'message'])`) because that option was never passed as form option. **After** this change, you could use Symfony Form `help` option and it will work as expected.